source/content/guides/disaster-recovery/03-site-goes-down.md

```diff
@@ -19,18 +19,18 @@ integration: [--]
 
 ## Open a Support Ticket
 
-In cases of downtime or significant functional failure in the Live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (included for all Enterprise contract customers), our support engineers will be the ones diagnosing the cause of downtime and working to get your site back up, and a ticket is the fastest way to get them up to speed and engaged.
+In cases of downtime or significant functional failure in the live environment, the first step is to open a support ticket. Even if you escalate the incident to your dedicated Customer Success Manager (available to all Enterprise contract customers), our engineering team will be the ones diagnosing the cause of the downtime and working to restore your site. Opening a ticket is the fastest way to get them up to speed and engaged.
 
-* **Diamond** and **Platinum** Account customers can report and escalate site downtime by clicking **Trigger Pantheon On-Call** from the Support tab.
-  * In cases where the dashboard is inaccessible, a ticket can be filed using a telephone ticketing service, accessible at **1(866)415-7624**. Note that this is strictly for filing a ticket, and you will not reach a support engineer by using this method.
-* **Gold** customers should click **Open Ticket** to open a support interaction to report site downtime.
-* **Silver** customers should click **Launch Chat**.
+* **Diamond** and **Platinum** customers can open an **emergency ticket** to report and escalate business-critical site downtime, from the Support tab in the workspace of the affected site.
+  * In cases where the dashboard is inaccessible, a support ticket can be filed using a telephone ticketing service, accessible at 1(866)415-7624. Note that this is strictly for filing a ticket, and you will not reach our engineering team by using this method.
+* **Gold** customers can open a **general support ticket** to report site downtime.
+* **Silver** customers can start a **live chat** to report site downtime.
```

Comment:
The changes we have made are for the Support page in the new dashboard experience (Phoebe) not the old dashboard experience that is screenshotted above. This page has not been updated.
To see screenshots for the Support page in the new dashboard, please see the engineering PR at https://github.com/pantheon-systems/dashboard/pull/3240

Reply:
Should we make it a task to update the docs once the `Site Dashboard -> Support` page does get upgraded to Phoebe? If we use language that isn't implemented yet, it causes a disconnect between what we tell the customer to do and what they're actually able to accomplish.

Reply:
As long as customers can still access the Phoebe pages and see the on-call verbiage, we should get those updated too. 

Chat --> Live chat
Launch chat --> Start chat
On-call / on-call help --> Emergency ticket, Open emergency ticket
Tickets --> General support ticket, Open general support ticket

For new changes, we use sentence case for headings and CTA labels. This aligns with the Design Systems. 

Suggest following the dashboard --> Support page changes, the phone number shouldn't be a text link. 

All changes for dashboard and Hermes copy are documented here for your reference: https://docs.google.com/document/d/1K5RodsnFHpoaM5nnDfV6mLd3iN8VdPeSY9YTCoqcNCM/edit

Reply:
Could I get permission to view that Google Doc?

Overnight, I realized that as a support agent, I cannot look at Phoebe for certain pages so what our customers see don't match what I see. My feedback is that we should update page verbiage to match the current state of what customers actually see. If a page has not been converted to Phoebe yet, then we should not convert its Docs language yet.

Reply:
Both interfaces need to be documented, I would suggest restructuring the page like this: 

<img width="1220" alt="Screenshot 2024-08-23 at 11 33 34 AM" src="https://github.com/user-attachments/assets/a0670043-3389-4017-ad23-be6b00bd8949">

Similar treatment for the whole page, each contact method an H2 containing subsections for each context. The doc structure should feel like this: 

```
## General support ticket
<usage and availability description>
### From the workspace 
<steps for this context> 
### From the site dashboard 
<steps for this context> 

## Live Chat
<usage and availability description>
### From the workspace 
<steps for this context> 
### From the site dashboard 
<steps for this context> 

## Call Us 
<usage and availability description>
### From the workspace 
<steps for this context> 
### From the site dashboard 
<steps for this context> 

## Emergency ticket
<usage and availability description>
### From the workspace 
<steps for this context> 
### From the site dashboard 
<steps for this context> 
```

Given the site support context will still say "On-Call" I think the release note copy needs to be reconsidered

Reply:
@stephpawlowski I recommend allowing a few more days to finalize the doc changes and the release note, would you be open to rescheduling at least this docs PR for next week instead of today?

Reply:
@stephpawlowski @IngridKwok @dylanbaumann 

I recommend the following: 

- [ ] Specify workspace support context above the bullet list  
- [ ] Replace the image with workspace support context instead (Phoebe) 
- [ ] Crosslink to the contact support page (which should be updated to cover both contexts)

I will own these todos but am considering the work blocked, [pending the resolution of this open discussion](https://github.com/pantheon-systems/documentation/pull/9191/files#r1729305038)

Reply:
there's another instance here where we're saying "engineering team", needs to be "support engineering team" or just "support team". The rest seems fine.

Reply:
I used a placeholder image for now, waiting on a new image - requested in slack thread. Otherwhise this feedback has been addressed by this commit: https://github.com/pantheon-systems/documentation/pull/9191/commits/9727a6cf122638b2e3c01176c81f367434fa6e9e